src/mapgen_v7.cpp

```diff
@@ -400,7 +407,7 @@ Biome *MapgenV7::getBiomeAtPoint(v3s16 p)
 	return bmgr->getBiome(heat, humidity, groundlevel);
 }
 
-//needs to be updated
+
```

Comment:
Why'd you remove this comment, seeing as how mountains now include base terrain?